features/shared-memory.yml

```
@@ -0,0 +1,27 @@
+name: SharedArrayBuffer and Atomics
+description: The `SharedArrayBuffer` object represents bytes shared between multiple workers and the main thread. The `Atomics` object safely accesses `SharedArrayBuffer` data to make sure predictable values are read and written and that operations are not interrupted."
```

Typo fix, I'll apply this myself and merge:

```suggestion
description: The `SharedArrayBuffer` object represents bytes shared between multiple workers and the main thread. The `Atomics` object safely accesses `SharedArrayBuffer` data to make sure predictable values are read and written and that operations are not interrupted.
```